website/source/guides/index.html.md

```diff
@@ -0,0 +1,16 @@
+---
+layout: "guides"
+page_title: "Guides"
+sidebar_current: "guides-home"
+description: |-
+  Welcome to the Terraform guides! The guides provide examples for common
```

Comment:
I think it only reads weird because at time of writing there is only 1. When there are multiple (assuming there will be) it reads fine I think.